TTS/auto_tts/model_hub.py

```diff
@@ -0,0 +1,530 @@
+import os
+
+from recipes.ljspeech.fast_pitch.train_fast_pitch import config as fastpitch_config
+from recipes.ljspeech.glow_tts.train_glowtts import config as glowtts_config
+from recipes.ljspeech.hifigan.train_hifigan import config as hifigan_config
+from recipes.ljspeech.multiband_melgan.train_multiband_melgan import config as multiband_melgan_config
+from recipes.ljspeech.univnet.train import config as univnet_config
+from recipes.ljspeech.vits_tts.train_vits import config as vits_config
+from recipes.ljspeech.wavegrad.train_wavegrad import config as wavegrad_config
+from recipes.ljspeech.wavernn.train_wavernn import config as waverrn_config
+from TTS.auto_tts.utils import pick_forwardtts_decoder, pick_forwardtts_encoder, pick_glowtts_encoder
+from TTS.trainer import TrainingArgs
+from TTS.tts.configs.glow_tts_config import GlowTTSConfig
+from TTS.tts.configs.tacotron2_config import Tacotron2Config
+from TTS.tts.models.forward_tts import ForwardTTSArgs
+from TTS.utils.manage import ModelManager
+
+# os.environ["CUDA_VISIBLE_DEVICES"] = "1"
+
+
+class TtsModels:
+    def __init__(
+        self,
+        batch_size: int,
+        mixed_precision: bool,
+        learning_rate: float,
+        epochs: int,
+        output_path: str = os.path.dirname(os.path.abspath(__file__)),
+    ):
+
+        self.batch_size = batch_size
+        self.output_path = output_path
+        self.mixed_precision = mixed_precision
+        self.learning_rate = learning_rate
+        self.epochs = epochs
+        self.manager = ModelManager()
+
+    def _single_speaker_from_pretrained(self, model_name):
```

Comment:
It'd be easier to use the ModelManager to parse the models from .models.json  so we dont need to add models manually